main/windows/systemTray.ts

```diff
@@ -0,0 +1,57 @@
+import { app, Menu, Tray as ElectronTray } from 'electron'
+import path from 'path'
+import store from '../store'
+
+let electronTray: ElectronTray
+
+export const setContextMenu = (
+  type: string,
+  clickHandlers: { show: () => void; hide: () => void },
+  displaySummonShortcut: boolean = store('main.shortcuts.altSlash')
+) => {
+  const separatorMenuItem = {
+    label: 'Frame',
+    click: () => {},
+    type: 'separator'
+  }
+  const menuItemLabelMap = {
+    hide: 'Dismiss',
+    show: 'Summon'
+  }
+  const label = menuItemLabelMap[type as keyof typeof menuItemLabelMap]
+  const actionMenuItem: Electron.MenuItemConstructorOptions = {
+    label,
+    click: () => clickHandlers[type as keyof typeof clickHandlers](),
+    toolTip: `${label} Frame`
+  }
+  const quitMenuItem = {
+    label: 'Quit',
+    click: () => app.quit()
+  }
+
+  if (displaySummonShortcut) {
```

Comment:
what does this do? I commented this out and it didnt seem to change anything but I might have missed a test case

Reply:
Registers the accelerator or not depending on whether the keyboard shortcut to summon / hide is enabled

Reply:
right but what does that do?

Reply:
It shows the system tray shortcut in the menu items, on Mac at least.  Ubuntu doesn't show them regardless, would be nice if the Electron docs mentioned this...

Reply:
yeah I see now, that's why I wasn't seeing it. makes sense 👌🏻

Reply:
https://www.electronjs.org/docs/latest/api/menu-item#menuitemaccelerator